index.js

```diff
@@ -41,6 +41,61 @@ function ELKReporter(runner) {
   });
 
   runner.on('end', function() {
+      /*
+      * When Cypress test passes, it provides a test instance to add context using mochawesome/addContext.
+      * But when Cypress test fails, it does not provide test instance to add context.
+      * However it provides test instance when that specific test finishes.
+      * So to add context on test failure, replace test objects in 'failures' array with its equivalent object from 'tests' array.
+      * 
+      * 1. Iterate over failures array. Push failed tests with no context in a map by mapping title to test's index in failures array.
```

Comment:
can we add the context information when the test fail or pass rather than needing to add them after all tests have executed?

Reply:
got it thanks